charts/signoz/values.yaml

```diff
@@ -1558,3 +1574,93 @@ otelCollectorMetrics:
           receivers: [prometheus]
           processors: [batch]
           exporters: [clickhousemetricswrite]
+
+# Default values for k8s-infra.
+# For complete list of configurations, check `values.yaml` of `k8s-infra` chart.
+# @ignored
+k8s-infra:
+  # -- Whether to enable K8s infra monitoring
+  enabled: true
+
+  # -- Endpoint/IP Address of the SigNoz.
+  otelCollectorEndpoint: my-release-signoz-otel-collector.platform.svc.cluster.local:4317
+
+  # Default values for OtelAgent
+  otelAgent:
+    name: "otel-agent"
+    image:
+      registry: docker.io
+      repository: otel/opentelemetry-collector-contrib
+      tag: 0.55.0
+      pullPolicy: IfNotPresent
+    imagePullSecrets: []
+
+    # OpenTelemetry Collector executable
+    command:
+      # -- OtelAgent command name
+      name: /otelcol-contrib
+      # -- OtelAgent command extra arguments
+      extraArgs: []
+
+    configMap:
+      # -- Specifies whether a configMap should be created (true by default)
+      create: true
+
+    # OtelAgent service
+    service:
+      # -- Annotations to use by service associated to OtelAgent
+      annotations: {}
+      # -- Service Type: LoadBalancer (allows external access) or NodePort (more secure, no extra cost)
+      type: ClusterIP
+
+    # -- Configure resource requests and limits. Update according to your own use
+    # case as these values might not be suitable for your workload.
+    # ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    # @default -- See `values.yaml` for defaults
+    resources:
+      requests:
+        cpu: 100m
+        memory: 100Mi
+      limits:
+        cpu: 1000m
+        memory: 1Gi
+
+  # Default values for OtelDeployment
+  otelDeployment:
+    name: "otel-deployment"
+    image:
+      registry: docker.io
+      repository: otel/opentelemetry-collector-contrib
+      tag: 0.55.0
+      pullPolicy: IfNotPresent
+    imagePullSecrets: []
+
+    # OpenTelemetry Collector executable
+    command:
+      # -- OtelDeployment command name
+      name: /otelcol-contrib
+      # -- OtelDeployment command extra arguments
+      extraArgs: []
+
+    configMap:
+      # -- Specifies whether a configMap should be created (true by default)
+      create: true
+
+    # OtelDeployment service
+    service:
+      # -- Annotations to use by service associated to OtelDeployment
+      annotations: {}
+      # -- Service Type: LoadBalancer (allows external access) or NodePort (more secure, no extra cost)
+      type: ClusterIP
+
+    # -- Configure resource requests and limits. Update according to your own use
+    # case as these values might not be suitable for your workload.
+    # ref: http://kubernetes.io/docs/user-guide/compute-resources/
+    # @default -- See `values.yaml` for defaults
+    resources:
+      requests:
+        cpu: 100m
+        memory: 100Mi
+      limits:
```

Comment:
@prashant-shahi since deployment behaves mostly as prometheus scraper part, more resources might be needed.